modules/HomePage/components/DeleteConfirmationModal/DeleteConfirmationModal.types.ts

```diff
@@ -0,0 +1,5 @@
+export interface DeleteConfirmationModalProps {
```

Comment:
```suggestion
export interface IDeleteConfirmationModalProps {
```

Best to get into the habit of prepending interfaces with I and types with T as it is the convention that sazim follows.